go/engine/identify2_test.go

```diff
@@ -1100,26 +1100,31 @@ func TestResolveAndCheck(t *testing.T) {
 	evilResolver := evilResolver{goodResolver, "t_alice", tracyUID}
 
 	var tests = []struct {
-		s string
-		e error
-		r libkb.Resolver
+		s       string
+		e       error
+		useEvil bool
 	}{
-		{"tacovontaco@twitter+t_tracy@rooter", nil, nil},
-		{"tacovontaco@twitter+t_tracy@rooter+t_tracy", nil, nil},
-		{"t_tracy", nil, nil},
-		{"t_tracy+" + string(tracyUID) + "@uid", nil, nil},
-		{"tacovontaco@twitter+t_tracy@rooter+foobunny@github", libkb.UnmetAssertionError{}, nil},
-		{"foobunny@github", libkb.ResolutionError{}, nil},
-		{"foobunny", libkb.NotFoundError{}, nil},
-		{"foobunny+foobunny@github", libkb.NotFoundError{}, nil},
-		{"t_alice", libkb.UIDMismatchError{}, &evilResolver},
-		{"t_alice+t_tracy@rooter", libkb.UnmetAssertionError{}, &evilResolver},
-		{"t_alice+" + string(aliceUID) + "@uid", libkb.UnmetAssertionError{}, &evilResolver},
+		{"tacovontaco@twitter+t_tracy@rooter", nil, false},
+		{"tacovontaco@twitter+t_tracy@rooter+t_tracy", nil, false},
+		{"t_tracy", nil, false},
+		{"t_tracy+" + string(tracyUID) + "@uid", nil, false},
+		{"tacovontaco@twitter+t_tracy@rooter+foobunny@github", libkb.UnmetAssertionError{}, false},
+		{"foobunny@github", libkb.ResolutionError{}, false},
+		{"foobunny", libkb.NotFoundError{}, false},
+		{"foobunny+foobunny@github", libkb.NotFoundError{}, false},
+		{"t_alice", libkb.UIDMismatchError{}, true},
+		{"t_alice+t_tracy@rooter", libkb.UnmetAssertionError{}, true},
+		{"t_alice+" + string(aliceUID) + "@uid", libkb.UnmetAssertionError{}, true},
+		// NOTE: Generic proofs are a WIP, this should change to a
+		// NotFoundError and a success case as the implementation proceeds
+		{"foobunny@mastodon.social", libkb.AppStatusError{}, false},
+		// TODO set this up for success!
+		{"t_alice@mastodon.social", libkb.AppStatusError{}, false},
```

Comment:
seems easy to forget about this, is there a ticket number that could be written next to the todo?

Reply:
nice thing is the test will fail since the expected status will change once it's fix but down for adding it

Reply:
ok if you're sure it won't get left sounds good